docker/Dockerfile

```diff
@@ -103,7 +103,7 @@ RUN cargo pgrx package --pg-config "/usr/lib/postgresql/${PG_VERSION_MAJOR}/bin/
 
 FROM builder AS builder-pgvector
 
-ARG PG_VERSION_MAJOR=16
+ARG PG_VERSION_MAJOR=17
```

Comment:
You're missing upgrading to 17 in the `pg_ivm` section. I wonder if it's enough to just add `ARG PG_VERSION_MAJOR` and it will retrieve 16 from the top-level one, so we don't need to constantly keep them all updated?

Reply:
Oh I forgot to add other changed lines into git.